src/main/java/design/MazeDesignerInteractor.java

```diff
@@ -0,0 +1,47 @@
+package design;
+
+import entities.DesignableMaze;
+
+public class MazeDesignerInteractor implements MazeDesignerInputBoundary {
+
+    DesignableMaze dm;
+
+    public void newMaze(){
+        dm = new DesignableMaze(11, 17);
+    }
+
+    public void resetMaze(){
+        dm.emptySetup();
+    }
+
+    public void buildWall(int row, int col){
+        dm.placeWall(row, col);
+    }
+    public void removeWall(int row, int col){
+        if (row != 0 && col != 0 && row != dm.getNumRow()-1 && col != dm.getNumCol()-1) {
+            dm.deleteWall(row, col);
+        }
+    }
+    public void startPoint(int row, int col){
+        dm.placeStart(row, col);
+    }
+    public void endPoint(int row, int col){
+        dm.placeEnd(row, col);
+    }
+    public void randomMaze(){
+        RandomizedPrim mg = new RandomizedPrim();
+        mg.generate(dm);
+        startPoint(1,1);
+        endPoint(getRows()-2, getCols()-2);
+    }
+    public int getRows(){
+        return dm.getNumRow();
+    }
+    public int getCols(){
+        return dm.getNumCol();
+    }
+    public char cellChar(int row, int col){
+        return dm.getCell(row, col);
+    }
+
```

Comment:
Don't see any methods from the out boundary being called here but I think it would be best to otherwise how does the information get sent back to the user?

Reply:
I agree, please refer to my comment on storing the output boundary as an instance attribute [here](https://github.com/CSC207-2022F-UofT/course-project-wasted-potential/pull/16/files#r1028497913). 

Shouldn't we be calling functions on the outputboundary (presenter) to actually view the results any interactions?